README.md

```diff
@@ -1,4 +1,79 @@
-### docker-pg-backup
+# docker-pg-backup
+
+This container can be utilized to create base backups for a 
+[postgres](https://www.postgresql.org/) instance which is configured to do
+[continuous archiving](https://www.postgresql.org/docs/9.2/continuous-archiving.html).
+
+## Usage
+
+### psql
+
+The container can be deployed beside the postgres instance. The postgres 
+data directory must be mounted to /pgdata. To configure psql inside the 
+backup container following environment variables must be set.
+
+* PGUSER 
+* PGHOST
+
+The password can either be specified via a 
+[pgpass](https://www.postgresql.org/docs/current/libpq-pgpass.html) file 
+or the PGPASSWORD variable. The entrypoint script is looking for a pgpass
+file in /run/secrets/pgpass and copies it into the right directory if it
+exists. For further information please look at the postgres documentation:
+
+* [Environment Variables](https://www.postgresql.org/docs/current/libpq-envars.html) 
+* [pgpass](https://www.postgresql.org/docs/current/libpq-pgpass.html)
+
+### wal-e
+
+Wal-e needs a couple environment variables to work:
+
+* AWS_ACCESS_KEY_ID
+* AWS_SECRET_ACCESS_KEY
+* WALE_S3_PREFIX: s3://{bucketname}/{path to where backups should be placed}
+* WALE_S3_ENDPOINT: http+path://minio:9000 // only needed for non s3 urls
+
+At the moment only the packages for s3 are installed. Wal-e supports a couple other 
+providers which could be installed if needed. For further configuration options 
+please look [here](https://github.com/wal-e/wal-e)
+
+### ofelia
+
+The entrypoint can create a minimal ofelia config if needed. By setting
+BACKUP_SCHEDULE to a cron like [schedule](https://godoc.org/github.com/robfig/cron)
+a backup will be created according to it. If you'd also like to delete older
+backups you can set BACKUPS_TO_RETAIN to a non negative integer doing
+this will create a job which keeps the number of backups specified 
+and deletes the older ones.
+
+If you need a more specific setup you can mount your own ofelia.conf file at 
+/etc/ofelia.conf. If this file exists the entrypoint won't touch it.
+
+For further information about [ofelia](https://github.com/mcuadros/ofelia) please visit their github page.
+
+### restore
+
+To restore the db a couple of steps must be taken:
+
+* create a new empty postgres instance
+* mount the data directory into the backup container
+* make sure postgres isn't running
+* use wal-e backup-fetch /pgdata LATEST to restore the latest backup
+* place a recovery.conf file in the /pgdata directory
+* start postgres
+
+To make this all a bit easier a script is located at /scripts/restore.sh
+It needs the RESTORE_COMMAND variable to be set to the correct command to 
+restore the wal files. Which it uses to create the 
+[recovery.conf](https://www.postgresql.org/docs/9.2/continuous-archiving.html) 
+file and place it inside the /pgdata directory. The backup can be specified as 
+the first argument for the script. For example `/scripts/restore.sh LATEST`.
```

Comment:
we could probably put this into /usr/local/bin, this is easier for the user.

Reply:
good idea, I'll change the Dockerfile + README